examples/air-quality-sensor-app/silabs/include/AppEvent.h

```diff
@@ -18,39 +18,23 @@
  */
 
 #pragma once
-#include <cstdint>
 
-struct AppEvent;
-typedef void (*EventHandler)(AppEvent *);
+#include "BaseAppEvent.h"
 
-struct AppEvent
+struct AppEvent : public BaseAppEvent
 {
     enum AppEventTypes
     {
-        kEventType_Button = 0,
-        kEventType_Timer,
-        kEventType_AirQualitySensor,
+        kEventType_AirQualitySensor = BaseAppEvent::kEventType_Max + 1,
```

Comment:
I am not sure about the +1
kEventType_Max wouldn't be used and you use it to delimit the end of BaseAppEvent enum. 
It doesn't matter much tho

Reply:
It is just so that they never overlap, if used in the future.